libraries/AP_GyroFFT/AP_GyroFFT.cpp

```diff
@@ -374,7 +387,17 @@ void AP_GyroFFT::update()
     if (!_rpy_health.x && !_rpy_health.y) {
         _health = 0;
     } else {
-        _health = MIN(_global_state._health, _ins->get_num_gyro_dynamic_notches());
+        uint8_t num_notches = 0;
+        for (auto &notch : _ins->harmonic_notches) {
+            if (notch.params.enabled()) {
+                num_notches = MAX(num_notches, notch.num_dynamic_notches);
+            }
+        }
+        if (num_notches == 0) {
+            // allow for operation with notches disabled
+            num_notches = 2;
```

Comment:
This should be 1. It would be easier just to make the initial value 1 and then you could remove this check.
See the original values calculated in AP_InertialSensor::init

Reply:
changed